src/main/resources/assets/integrateddynamics/lang/en_us.json

```diff
@@ -1042,7 +1042,7 @@
     "operator.integrateddynamics.relational.gt": "Greater Than",
     "operator.integrateddynamics.relational.lt": "Less Than",
     "operator.integrateddynamics.relational.notequals": "Not Equals",
-    "operator.integrateddynamics.relational.notequals.info": "Type one and two must be equal.",
+    "operator.integrateddynamics.relational.notequals.info": "Type one and two must not be equal.",
```

Comment:
I suspect I made an error here. I guess this statement applies to the _type_ of the inputs, which must be the same for both equals and notequals operators.

Please confirm.

Reply:
Not just the type, but the actual *value* must be not equal.